tests/foreman/ui_airgun/test_location.py

```diff
@@ -27,3 +37,338 @@ def test_positive_create(session):
             'description': gen_string('alpha'),
         })
         assert session.location.search(loc_name)[0]['Name'] == loc_name
+
+
+def test_positive_delete(session):
+    loc_name = gen_string('alpha')
+    with session:
+        session.location.create({'name': loc_name})
+        session.location.delete(loc_name)
+        assert not session.location.search(loc_name)
+
+
+@run_only_on('sat')
+@tier2
+def test_positive_create_with_location_and_org(session):
+    """Create and select both organization and location.
+
+    :id: a1640ada-d4e9-447e-9e1b-40d17e1ede7e
+
+    :expectedresults: Both organization and location are selected.
+
+    :CaseLevel: Integration
+    """
+    loc_name = gen_string('alpha')
+    org = entities.Organization().create()
+    with session:
+        session.location.create({
+            'name': loc_name,
+            'description': gen_string('alpha'),
+        })
+        session.organization.select(org_name=org.name)
+        session.location.select(loc_name=loc_name)
+        assert session.location.search(loc_name)[0]['Name'] == loc_name
+        assert session.organization.search(org.name)[0]['Name'] == org.name
+
+
+@run_only_on('sat')
+@tier2
+def test_positive_update_subnet(session):
+    """Add/Remove subnet from/to location
+
+    :id: fe70ffba-e594-48d5-b2c5-be93e827cc60
+
+    :expectedresults: subnet is added and removed from the location
+
+    :CaseLevel: Integration
+    """
+    ip_addres = gen_ipaddr(ip3=True)
+    subnet = entities.Subnet(
+        network=ip_addres,
+        mask='255.255.255.0',
+    ).create()
+    loc = entities.Location().create()
+    with session:
+        session.location.update(
+            loc.name, {'subnets.resources.assigned': [subnet.name]})
+        loc_values = session.location.read(loc.name)
+        assert loc_values['subnets']['resources']['assigned'][0] == \
```

Comment:
probably move `subnet.name + " (" + ip_addres + "/24)"` outside in variable
`subnet_name = "{}({}/24)".format(subnet.name, ip_address)`
or even
`subnet_name = "{}({}/{})".format(subnet.name, ip_address, subnet.cidr)`